mohammad/JS/JS-basicSyntax/main.js

```diff
@@ -0,0 +1,101 @@
+// add number a to b
+function add(a, b) {
+  return a + b;
+}
+
+// flip a coin and log the result (head or tail)
+function flipACoin() {
+  if (Math.random() < 0.5) console.log("head");
+  else console.log("tail");
+}
+
+// concat two strings
+function concat(stringA, stringB) {
+  return stringA + stringB;
```

Comment:
this is ok, but there is another approach for concating strings which is used more

Reply:
Can you tell me what that one is?